datadog_checks_dev/datadog_checks/dev/tooling/trello.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, config):
             'Runtime-Security': '5f3148683b7428276f0f2133',
             'Infra-Integrations': '5f9f9e09af18c18c628d80ee',
             'Remote-Config': '619262c91ae65d40bafb576f',
+            'Agent-Metrics-Logs': '62a9bbeb1c71b2208581744e',
```

Comment:
We still have `'Core'` as an option for the `team_list_map`. Should this be removed now that these are added?

Reply:
Eventually this will get removed, but for at least one more release cycle we would want to keep it to avoid any troubles.